swap/swap.go

```diff
@@ -76,8 +76,8 @@ type Owner struct {
 // Params encapsulates param
 type Params struct {
 	LogPath             string // optional audit log path
-	PaymentThreshold    int64  // honey amount at which a payment is triggered
-	DisconnectThreshold int64  // honey amount at which a peer disconnects
+	PaymentThreshold    uint64 // honey amount at which a payment is triggered
```

Comment:
I don't agree to changing the data type here.

While I understand why you want to do it, the way you propose now has the consequence that now on every comparison we need to do a conversion, while as it was before (if I interpret correctly) the conversion is only done once, when we set these thresholds in the params.

Reply:
Changed back. Fully agree with your reasoning, thanks